bundles/org.openhab.ui.habot/src/main/resources/OH-INF/config/config.xml

```diff
@@ -6,7 +6,7 @@
 
 	<config-description uri="ui:habot">
 		<parameter name="useGzipCompression" type="boolean">
-			<label>Use compressed web app resources</label>
+			<label>Use Compression</label>
```

Comment:
Why this change? It's obscure enough already and this makes it even more vague :)
For that matter this parameter was more of an experiment and never worked too well anyways (for instance AFAIK it still doesn't work with myopenhab.org/OH Cloud). Perhaps we should just remove the config description and simply keep it as an undocumented feature - that would remove the HABot entry from the configurable services.

Reply:
It's because the new SAT version checks the label length and these 32 chars exceed the 25 chars limit (https://github.com/openhab/static-code-analysis/pull/360) so without fixing or supressing it the build would fail. Since this was the only change required, I reduced the length as it has a good description. I can also remove it if that makes you even more happy. :slightly_smiling_face:

Reply:
Ah, that explains it! 👍  I really thought it was an accidental, unrelated change :)
If you don't maybe it's indeed time to remove ui:habot config-description. This parameter was confusing people and if it's ever needed it can be set via the console or other means.

Reply:
OK it's gone now. :-)